pkg/bindings/images/build.go

```diff
@@ -292,7 +292,7 @@ func Build(ctx context.Context, containerFiles []string, options entities.BuildO
 	} else if len(platform) > 0 {
 		platform += "/" + runtime.GOARCH
 	}
-	if len(platform) > 0 {
+	if len(platform) > 0 && platform != "/" {
```

Comment:
Future maintainers are going to be baffled here (as I am right now): who/what/where is setting platform to "/", and why? Why do we have to special-case this? Why can't it be fixed at the source? If it can't be fixed at the source (setting platform to a nice value like nil or ""), at least a comment here might be welcome by the next person who has to dive into this code. Same, below.

Reply:
Yes let me add a comment or rather convert it to make sure that `platform` is empty instead of `/`, issue is happening because if you see above the platform  string is using `/` as concat while forming the complete platform strings and when fields are empty it just ends up with a blank `/`.

Reply:
But this is not true here. In the other case below maybe but not here.
There is no way the platform string ends up with just `/`, it explicitly checks for `len(options.OS) > 0` and `len(options.Architecture) > 0` so there is no way that can be just empty.

Reply:
@Luap99 ah yes my bad its about the check which is below this, not this one.

Reply:
Nvm I think i have a better fix for this.